New training page

patricia-ternes opened the pull request:
Note: I did this on Grasi's laptop, so it is her name on commits, sorry about it.

This pull request creates a new training page.
How it works:

1. The `pages/training.md` was updated to retrieve information from `_data/events.yml`
2. For each event in `_data/events.yml` a banner will be created using the template available in `_includes/event.html` (and the style defined in `_sass/_12_events-calendar.scss`).
  - Note that the information under each event in `_data/events.yml` is minimum because I am using the course `code` to retrieve the course information from `_data/training_courses.yml`.
  - DataVis has no information under the `_data/training_courses.yml`, so one of the banners is empty, we can fix this later on.
3. The `pages/training.md` code verifies the date and display only future events. So in theory, it is not necessary delete old events. But I am not sure how this will be handled in future by the GH action.

This PR override the PR #223, so if this one is approved, the #223 should be deleted.

jhodrien commented on assets/css/styles_feeling_responsive.scss:
```diff
@@ -47,3 +47,5 @@ sitemap:
 
 @import "11_syntax-highlighting.scss";
+
+@import "12_events-calendar"
```
Inconsistent with previous import lines by ommiting the .scss.  Missed end of line character.

jhodrien reviewed (commented):
I like what you end up with, but it feels like this replaces the calendar page more than it does the training page.  So I think we need to tidy that up at the same time as this gets committed.

patricia-ternes commented:
As we discussed, the calendar page was not necessary anymore, so I removed.
I also removed the Courses page as this is not updated and the information may lead to wrong assumptions. I will create a Issue related to this. So we can accept/close this PR and handle the Courses page in other moment.

I will also create a Issue for the DataVis course description.
